clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -49,9 +51,22 @@ const ScanResultsForm = () => {
   const [selectedColumns, setSelectedColumns] =
     useState<ColumnMetadata[]>(ALL_COLUMNS);
 
+  const { successAlert } = useAlert();
+  const { handleError } = useAPIHelper();
+
   const confirmRegisterSelectedSystems = async () => {
     dispatch(chooseSystemsForReview(selectedSystems.map((s) => s.fides_key)));
-    await upsertSystems(selectedSystems);
+    const response = await upsertSystems(selectedSystems);
+
+    if (isErrorResult(response)) {
+      return handleError(response.error);
+    }
+
+    successAlert(
```

Comment:
Note: need to resolve same issue here as in 239 where the toast might not show up across nextjs zones